sdk/cosmos/azure-cosmos/azure/cosmos/_retry_utility.py

```diff
@@ -95,7 +103,16 @@ def Execute(client, global_endpoint_manager, function, *args, **kwargs):
             client.last_response_headers[
                 HttpHeaders.ThrottleRetryWaitTimeInMs
             ] = resourceThrottle_retry_policy.cumulative_wait_time_in_milliseconds
-
+            # If container does not have throughput, results will return empty list, but to align with other SDKs
+            # we manually raise a 404. We raise it here, so we can handle it in retry utilities
+            if result and isinstance(result[0], dict) and 'Offers' in result[0] and \
+                    not result[0]['Offers'] and args[3].method == 'POST':
+                # Grab the link used for getting throughput properties to add to message.
+                link = ast.literal_eval(args[3].body)["parameters"][0]["value"]
```

Comment:
Same comment here - what string value is this such that json.loads wouldn't work?